packages/react/src/components/Pagination/Pagination.module.css

```diff
@@ -0,0 +1,75 @@
+.pagination {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.whitespace {
+  padding-right: var(--component-button-space-padding-x-small);
+  padding-left: var(--component-button-space-padding-x-small);
```

Comment:
That's the only tokens that exist for spacing and the ones that the figma design says I should use

Reply:
Maybe we could try a different solution for this. Instead of making a "whitespace" div. We can just use the `Button` and toggle `vibility: hidden;` on it instead. This way it will also always be the correct size.

This would also probably let us use flex gap instead of left/right margins 🤔

Reply:
Yes I thought of doing that at first, I was only wondering if it was bad practise to use elements just to make them invisible. 

Muligens er dette bare et valg mellom 2 onder og da er kanskje det best å bare ta i bruk button og gjøre de hidden ja

Reply:
Button blir jo synlig i domen men det er kanskje ikke så farlig

Reply:
Ja, den tar opp plass som vi ønsker, men "skjult" fra alt annet. Så når vi trenger å vise den så er det bare å fjerne klassen :) 

https://developer.mozilla.org/en-US/docs/Web/CSS/visibility